app/assets/stylesheets/posts/_post.scss

```diff
@@ -46,7 +46,8 @@
 .user-email {
   display: flex;
   width: fit-content;
-  color: $tomato !important;
+  color: black !important;
+  font-weight: 500;
```

Comment:
will be cool to sort alphabetically this block and use a color from `colors.scss`

Reply:
thanks for the observation, now should be fixed!